src/fetch/index.t-test.ts

```
@@ -1,8 +1,34 @@
-import FetchT from "./index";
+import {
+  AsJsonApi,
+  DefineApiEndpoints,
+  ExcessiveQueryError,
+  MissingQueryError,
+} from "../core";
+import FetchT, { ValidateUrl } from "./index";
 import JSONT from "../json";
+import { Equal, Expect } from "../core/type-test";
+import { C } from "../compile-error-utils";
 const JSONT = JSON as JSONT;
```

Biome

[error] Shouldn't redeclare 'JSONT'. Consider to delete it or rename it.

'JSONT' is defined here:


(lint/suspicious/noRedeclare)